drivers/timer/nrf_rtc_timer.c

```diff
@@ -667,6 +674,15 @@ static int sys_clock_driver_init(const struct device *dev)
 			CLOCK_CONTROL_NRF_LF_START_AVAILABLE :
 			CLOCK_CONTROL_NRF_LF_START_STABLE);
 
+#if defined(CONFIG_INIT_ARCH_HW_AT_BOOT)
```

Comment:
This is not valid, this Kconfig is for setting the default architecture peripherals and registers to known values, architecture in this case is ARMv7E-M - the ARM architecture has no knowledge of the nordic rtc peripheral because that is not part of the ARM architecture, and it would be wrong to start adding SoC-specific handling to this to address an issue with 1 bootloader.

Reply:
Thanks! So, would it be right is I just remove the #if directive and change the commit message accordingly?